projects/1.21/assets/psi/psi/lang/zh_cn.json

```
@@ -0,0 +1,1123 @@
+  "psi.spellpiece.operator_vector_cross_product": "运算符：矢量叉乘",
+  "psi.spellpiece.operator_vector_cross_product.desc": "A x B",
+  "psi.spellpiece.operator_vector_normalize": "运算符：矢量归一",
+  "psi.spellpiece.operator_vector_normalize.desc": "将矢量归一化，即令其模长变为1",
+  "psi.spellpiece.operator_vector_negate": "运算符：相反矢量",
+  "psi.spellpiece.operator_vector_negate.desc": "求矢量的相反矢量",
+  "psi.spellpiece.operator_vector_magnitude": "运算符：矢量模长",
+  "psi.spellpiece.operator_vector_magnitude.desc": "获取矢量的模长",
+  "psi.spellpiece.operator_vector_construct": "运算符：矢量构造",
+  "psi.spellpiece.operator_vector_construct.desc": "使用XYZ分量构造矢量",
+  "psi.spellpiece.operator_vector_extract_x": "运算符：提取矢量X",
+  "psi.spellpiece.operator_vector_extract_x.desc": "提取矢量的X坐标",
+  "psi.spellpiece.operator_vector_extract_y": "运算符：提取矢量Y",
+  "psi.spellpiece.operator_vector_extract_y.desc": "提取矢量的Y坐标",
+  "psi.spellpiece.operator_vector_extract_z": "运算符：提取矢量Z",
+  "psi.spellpiece.operator_vector_extract_z.desc": "提取矢量的Z坐标",
+  "psi.spellpiece.operator_vector_project": "运算符：矢量投影",
+  "psi.spellpiece.operator_vector_project.desc": "将矢量A投影至矢量B",
+  "psi.spellpiece.operator_vector_dot_product": "运算符：矢量点乘",
+  "psi.spellpiece.operator_vector_dot_product.desc": "A . B",
+  "psi.spellpiece.operator_closest_to_point": "运算符：离点最近",
+  "psi.spellpiece.operator_closest_to_point.desc": "从列表中选出距离指定位置最近的实体",
+  "psi.spellpiece.operator_random_entity": "运算符：随机实体",
+  "psi.spellpiece.operator_random_entity.desc": "从列表中随机选择实体",
+  "psi.spellpiece.operator_list_add": "运算符：加入列表",
+  "psi.spellpiece.operator_list_add.desc": "将目标实体加入列表",
+  "psi.spellpiece.operator_list_remove": "运算符：移出列表",
+  "psi.spellpiece.operator_list_remove.desc": "从列表中移除目标实体",
+  "psi.spellpiece.operator_planar_normal_vector": "运算符：平面法矢量",
+  "psi.spellpiece.operator_planar_normal_vector.desc": "通过给定轴向矢量求垂直矢量，等价于绕<1,1,1>旋转2π/3",
+  "psi.spellpiece.operator_vector_rotate": "运算符：旋转矢量",
+  "psi.spellpiece.operator_vector_rotate.desc": "绕给定轴旋转矢量",
+  "psi.spellpiece.operator_gamma_function": "运算符：伽玛函数",
+  "psi.spellpiece.operator_gamma_function.desc": "Γ(x)，等价于(x-1)!",
+  "psi.spellpiece.operator_list_size": "运算符：列表大小",
+  "psi.spellpiece.operator_list_size.desc": "获取列表中的实体项数",
+  "psi.spellpiece.operator_list_union": "运算符：合并列表",
+  "psi.spellpiece.operator_list_union.desc": "将两个列表合并成一个",
+  "psi.spellpiece.operator_list_intersection": "运算符：列表求交",
+  "psi.spellpiece.operator_list_intersection.desc": "返回列表，其中仅包含两列表中均有的表项",
+  "psi.spellpiece.operator_list_exclusion": "运算符：列表求差",
+  "psi.spellpiece.operator_list_exclusion.desc": "返回列表，其中仅包含属于第一个列表但不属于第二个列表的表项",
+  "psi.spellpiece.operator_list_index": "选择符：索引元素",
+  "psi.spellpiece.operator_list_index.desc": "以指定索引从列表中获取实体",
+  "psi.spellpiece.operator_root": "运算符：开方",
+  "psi.spellpiece.operator_root.desc": "求数值的n次方根",
+  "psi.spellpiece.operator_extract_sign": "运算符：符号",
+  "psi.spellpiece.operator_extract_sign.desc": "获取输入值的符号",
+  "psi.spellpiece.operator_vector_extract_sign": "运算符：符号矢量",

```

```suggestion
  "psi.spellpiece.operator_vector_extract_sign": "运算符：矢量符号",
```
从后面看应该是Operator: Vector Signum